examples/vnet/doca_gw.c

```diff
@@ -81,15 +82,15 @@ struct doca_gw_port * doca_gw_port_start(struct doca_gw_port_cfg *cfg, struct do
 		//TODO: need to parse devargs
 		DOCA_LOG_INFO("new doca port type:dpdk port id:%s", cfg->devargs);
 		port = doca_gw_dpdk_port_start(cfg, err);
-		if (port)
```

Comment:
not clear to me why you removed it. doca dpdk user want to say this doca_port is dpdk_port 0 (SW have to know this mapping).

Reply:
i move it to the last of this function, only code optimization.
have updated.